src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1546,7 +1545,7 @@ export class PostgresStorageAdapter {
       }
     }
 
-    const qs = `SELECT ${columns} FROM $1:name ${wherePattern} ${sortPattern} ${limitPattern} ${skipPattern} ${groupPattern}`;
```

Comment:
Are you sure you do not need to join the column there? Or is it the default for ES6 template strings when it comes to arrays?

Reply:
Did I reject this? Sorry, I'm not quite sure what I'm doing when it comes to reviews :)

Reply:
I did join the columns here. They weren't joined before.